core/jazz-ui/src/app/secondary-components/create-service/oss/create-service.component.html

```diff
@@ -65,6 +65,25 @@ <h4 class="bold-title">
                 </div>
             </div>
         </section>
+        <section class="form-row deployment-targets" *ngIf="typeOfService === 'api' && buildEnvironment.createService.apiDeploymentTargets">
+          <div class="row">
+            <div class="col-lg-4 col-md-4">
+              <p class="step-title">Choose your API deployment target<span>*</span></p>
+            </div>
+            <div class="col-lg-8 col-md-8 radio-options">
+                <div class="radio-container" (click)="apiDeployment='aws_apigateway'">
+                  <input type="radio" name="target" value="aws_apigateway" [ngModel]="apiDeployment">
+                  <label><span class='outer'></span><span class="background"></span>AWS API Gateway</label>
+                </div>
+                <div class="radio-container" (click)="apiDeployment='gcp_apigee'">
```

Comment:
APIGEE UI component can be enabled only when APIGEE extension is enabled by admin. I will create a separate issue of this.